app/views/items/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>All Items</h1>
+<% @items.each do |item| %>
+  <section id= "item-<%= item.id %>">
+  <p><%= "Item: #{item.name}" %></p>
+  <p><%= "Price: $#{item.price}" %></p>
+  <p><%= "Supermarket Name: #{item.supermarket.name}" %></p>
```

Comment:
Also consider creating a method in the Item model for `supermarket_name` that holds this logic. I know that seems like a small change, but it would adhere more to the Law of Demeter. As a general rule, I try to only chain one method onto an object that is digging for info in my view. Here, you're digging into the item, then digging into the supermarket.